docs/configuration.md

```diff
@@ -714,6 +714,15 @@ Apart from these, the following properties are also available, and may be useful
   </td>
 </tr>
 <tr>
+  <td><code>spark.deploy.maxCoresPerExecutor</code></td>
```

Comment:
This will be `spark.executor.cores` instead (which is currently not already documented). The description would be something like:
"""
Default: 2 in YARN mode, all the available cores on the worker in standalone mode.

The number of cores to use on each executor. For YARN and standalone mode only.

In standalone mode, setting this parameter allows an application to run multiple executors on the same worker, provided that there are enough cores on that worker. Otherwise, only one executor per application will run on each worker.
"""